integration-tests/tests/interfaces_test.go

```diff
@@ -49,26 +49,32 @@ const (
 
 type interfaceSuite struct {
 	common.SnappySuite
-	snapPath, sampleSnap string
-	slot, plug           string
-	autoconnect          bool
+	snapPaths, sampleSnaps []string
+	slot, plug             string
+	autoconnect            bool
 }
 
 func (s *interfaceSuite) SetUpTest(c *check.C) {
 	s.SnappySuite.SetUpTest(c)
 
 	var err error
-	s.snapPath, err = build.LocalSnap(c, s.sampleSnap)
-	c.Assert(err, check.IsNil)
 
-	common.InstallSnap(c, s.snapPath)
+	s.snapPaths = make([]string, len(s.sampleSnaps))
+	for i := 0; i < len(s.sampleSnaps); i++ {
```

Comment:
If this is multiple snaps now, should  we switch to `SetUpTestSuite` (or what it is called in gocheck)?